src/amo/components/AddonMoreInfo.js

```diff
@@ -16,52 +17,48 @@ export class AddonMoreInfoBase extends React.Component {
   render() {
     const { addon, i18n } = this.props;
 
-    // TODO: Use the addonType for the privacy text, so it reads
-    // "for this extension", "for this theme", etc.
     return (
       <section className="AddonMoreInfo">
```

Comment:
If `Card` were a section you could pass it a `className` and set it in the component, then  this section can go away.